packages/big-design/src/components/Counter/spec.tsx

```diff
@@ -261,7 +262,7 @@ test('value increases and decreases with arrow keypresses', () => {
   const { getByDisplayValue } = render(counterMock(requiredAttributes));
 
   const counter = getByDisplayValue('5') as HTMLInputElement;
-  counter.focus();
+  act(() => counter.focus());
```

Comment:
Wrapping in `act` was easier fix, we could potentially update this file to use `userEvent`.

Reply:
Could you add a `TODO` comment here so we know to come back around to it?